Assets/Scripts/ScreenTapping.cs

```diff
@@ -9,14 +9,35 @@
 using UnityEngine.EventSystems;
 using System;
 
-public class ScreenTapping : MonoBehaviour, IPointerDownHandler
+public class ScreenTapping : MonoBehaviour, IPointerDownHandler, IBeginDragHandler, IDragHandler, IEndDragHandler
 {
     [SerializeField]
     [Tooltip("The prefab to instantiate for ScreenTapping.")]
     GameObject prefabScreenTapping;
 
     Vector2 localPoint;
```

Comment:
localPoint doesn't need to be a field. It can just be a local variable within a function.